### PR TITLE
Update readme.md of BC001 PS01 with rules identifiers

### PR DESCRIPTION
**Pull request by pjanck**

### CODE REVIEW

File: tests/PS01/readme.md [hunk 1]
```diff
@@ -80,11 +80,14 @@ Not covered by the IDS file (must be checked otherwise):
         - one `.LENGTHUNIT.` and
         - one `.PLANEANGLEUNIT.`.
 - (RI-03) An `IfcGeometricRepresentationContext` shall be associated with the `IfcProject`. The project origin should be `(0,0,0)`.
+- (RI-64) The placement of `IfcAnnotation` shall be relative to the `IfcFacility` in which it is contained.
+- (RI-65) The placement of `IfcFacility` shall be absolute within the `IfcGeometricRepresentationContext`.
 - (RI-77) `IfcAnnotation.USERDEFINED` with `ObjectType='Annotation point'` shall have `'Point'` geometry and use `IfcCartesianPoint`.
-- (RI-87) The model shall containts an IfcProjectedCRS that defines the map coordinate system and an IfcMapConversion or `IfcRigidOperation` that defines the mapping between `IfcProjectedCRS` and `IfcGeometricRepresentationContext`. 
+- (RI-87) The model shall containts an `IfcProjectedCRS` that defines the map coordinate system and an `IfcMapConversion` or `IfcRigidOperation` that defines the mapping between `IfcProjectedCRS` and `IfcGeometricRepresentationContext`.
```
Comment (evandroAlfieri) [inline tests/PS01/readme.md:86]: @apinzenoehler, this was suggested as a very good fit for a Gherkin rule (GRF002). Are you going to leave it as textual requirement or are you still planning to develop a rule for it?

File: tests/PS01/readme.md [hunk 2]
```diff
@@ -80,11 +80,14 @@ Not covered by the IDS file (must be checked otherwise):
         - one `.LENGTHUNIT.` and
         - one `.PLANEANGLEUNIT.`.
 - (RI-03) An `IfcGeometricRepresentationContext` shall be associated with the `IfcProject`. The project origin should be `(0,0,0)`.
+- (RI-64) The placement of `IfcAnnotation` shall be relative to the `IfcFacility` in which it is contained.
```
Comment (evandroAlfieri) [inline tests/PS01/readme.md:83]: Same as bullet 2 here https://github.com/buildingSMART/IFC4.x-IF/pull/48#discussion_r1260868058

File: tests/PS01/readme.md (hunk 1 above)
Comment (apinzenoehler) [reply tests/PS01/readme.md:86]: According to my memory this was already discussed between @aothms and @pjanck and is work in progress.

@aothms and @pjanck : is my memory correct?

File: tests/PS01/readme.md (hunk 2 above)
Comment (pjanck) [reply tests/PS01/readme.md]: Resolved in 90c60c1

File: tests/PS01/readme.md (hunk 1 above)
Comment (pjanck) [reply tests/PS01/readme.md:86]: Your memory is correct. I believe the final conclusion was, that this requirement imposes no additional requirements on the dataset as what is already defined by the schema.

We are, however, imposing a test plan specific requirement of the specific CRS in use - which was deemed out-of-scope for a Gherkin rule.

**Review by evandroAlfieri — APPROVED** [review]

**Pull request merged**